<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_article</class>
 <widget class="QWidget" name="add_article">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>713</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#add_article{
background: rgb(240, 237, 255);
border: none;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="5" column="5">
    <widget class="QLineEdit" name="lineEdit_name"/>
   </item>
   <item row="7" column="1" colspan="3">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_achat">
     <property name="suffix">
      <string> â‚¬</string>
     </property>
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="maximum">
      <double>1000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="11" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>55</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="10" column="3" colspan="4">
    <widget class="QLineEdit" name="lineEdit_path">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="5">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_vente">
     <property name="suffix">
      <string> â‚¬</string>
     </property>
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="maximum">
      <double>1000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="12" column="0" colspan="8">
    <widget class="QFrame" name="frame_2">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>150</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#frame_2 {
background: rgb(46, 54, 58);
border: none;
border-right: 1px solid black;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>153</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pushButton_add">
        <property name="text">
         <string>Ajouter</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>152</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="5">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>55</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">color:rgb(0,0,0);</string>
     </property>
     <property name="text">
      <string>RÃ©fÃ©rence article :</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="label_2">
     <property name="styleSheet">
      <string notr="true">color:rgb(0,0,0);</string>
     </property>
     <property name="text">
      <string>CatÃ©gorie :</string>
     </property>
    </widget>
   </item>
   <item row="5" column="7">
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>149</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" colspan="8">
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>150</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#frame {
background: rgb(46, 54, 58);
border: none;
border-right: 1px solid black;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
	color: rgb(255, 255, 255);
}</string>
        </property>
        <property name="text">
         <string>Ajouter un article</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QPushButton" name="pushButton_addImg">
     <property name="text">
      <string>ðŸ“Ž Joindre une photo</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>149</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="4">
    <widget class="QPushButton" name="pushButton_addCat">
     <property name="text">
      <string>+</string>
     </property>
    </widget>
   </item>
   <item row="10" column="2">
    <widget class="QPushButton" name="pushButton_delImg">
     <property name="text">
      <string>x</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="3">
    <widget class="QLineEdit" name="lineEdit_ref"/>
   </item>
   <item row="3" column="1" colspan="3">
    <widget class="QComboBox" name="comboBox_cat">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="4" column="5">
    <widget class="QLabel" name="label_6">
     <property name="styleSheet">
      <string notr="true">color:rgb(0,0,0);</string>
     </property>
     <property name="text">
      <string>Nom complet :</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLabel" name="label_3">
     <property name="styleSheet">
      <string notr="true">color:rgb(0,0,0);</string>
     </property>
     <property name="text">
      <string>Montant achat :</string>
     </property>
    </widget>
   </item>
   <item row="3" column="6">
    <widget class="QPushButton" name="pushButton_addVendor">
     <property name="text">
      <string>+</string>
     </property>
    </widget>
   </item>
   <item row="6" column="5">
    <widget class="QLabel" name="label_7">
     <property name="styleSheet">
      <string notr="true">color:rgb(0,0,0);</string>
     </property>
     <property name="text">
      <string>Montant vente :</string>
     </property>
    </widget>
   </item>
   <item row="2" column="5">
    <widget class="QLabel" name="label_5">
     <property name="styleSheet">
      <string notr="true">color:rgb(0,0,0);</string>
     </property>
     <property name="text">
      <string>Fournisseur :</string>
     </property>
    </widget>
   </item>
   <item row="3" column="5">
    <widget class="QComboBox" name="comboBox_vendor">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="11" column="5">
    <widget class="QLabel" name="label_idArticle">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboBox_cat</tabstop>
  <tabstop>pushButton_addCat</tabstop>
  <tabstop>comboBox_vendor</tabstop>
  <tabstop>pushButton_addVendor</tabstop>
  <tabstop>lineEdit_ref</tabstop>
  <tabstop>lineEdit_name</tabstop>
  <tabstop>doubleSpinBox_achat</tabstop>
  <tabstop>doubleSpinBox_vente</tabstop>
  <tabstop>pushButton_addImg</tabstop>
  <tabstop>pushButton_delImg</tabstop>
  <tabstop>lineEdit_path</tabstop>
  <tabstop>pushButton_add</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
